src/main/java/com/board/project/common/ExceptionAdvice.java

```diff
@@ -0,0 +1,21 @@
+package com.board.project.common;
+
+import javassist.NotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ExceptionAdvice {
+
+    @ExceptionHandler(NotFoundException.class)
+    public ApiResponse<String> NoSuchElementException(NotFoundException e) {
```

Comment:
메서드는 동사이며 카멜케이스로 작성해주셔야 합니다.
```suggestion
    public ApiResponse<String> noSuchElementException(NotFoundException e) {
```